pkg/storage/snapshot/source.go

```diff
@@ -147,9 +147,14 @@ func (s *vmSnapshotSource) Unlock() (bool, error) {
 	}
 
 	vmCopy.Status.SnapshotInProgress = nil
-	_, err = s.controller.Client.VirtualMachine(vmCopy.Namespace).UpdateStatus(context.Background(), vmCopy, metav1.UpdateOptions{})
+	vmCopy, err = s.controller.Client.VirtualMachine(vmCopy.Namespace).UpdateStatus(context.Background(), vmCopy, metav1.UpdateOptions{})
```

Comment:
there is a comment: https://github.com/kubevirt/kubevirt/pull/12601/files#diff-98499752dcfec117369933a187ef821e70589d979cba2b3b5cdfa1d9149be198R113-R115
That says UpdateStatus does not return the updated resource. Is it not true any? or is there any other value in putting the return value to vmCopy and then in s.vm=vmCopy couple of lines after

Reply:
`updateStatus` returns the resource our "status updater" (pkg/util/status) abstraction does not so maybe the comment can be deleted